eva-accession-ws/src/test/java/uk/ac/ebi/eva/accession/ws/ClusteredVariantsRestControllerTest.java

```diff
@@ -0,0 +1,134 @@
+public class ClusteredVariantsRestControllerTest {
+
+    private static final String URL = "/v1/clustered-variants/";
+
+    @Autowired
+    private DbsnpClusteredVariantAccessioningRepository repository;
+
+    @Autowired
+    private ClusteredVariantsRestController controller;
+
+    @Autowired
+    private TestRestTemplate testRestTemplate;
+
+    private Iterable<DbsnpClusteredVariantEntity> generatedAccessions;
+
+    @Before
+    public void setUp() {
+        repository.deleteAll();
+
+        ClusteredVariant variant1 = new ClusteredVariant("ASMACC01", 1101, "CHROM1", 1234, VariantType.SNV, false);
+        ClusteredVariant variant2 = new ClusteredVariant("ASMACC02", 1102, "CHROM2", 1234, VariantType.MNV, false);
+
+        DbsnpClusteredVariantSummaryFunction function = new DbsnpClusteredVariantSummaryFunction();
+        DbsnpClusteredVariantEntity entity1 = new DbsnpClusteredVariantEntity(1L, function.apply(variant1), variant1);
+        DbsnpClusteredVariantEntity entity2 = new DbsnpClusteredVariantEntity(2L, function.apply(variant2), variant2);
+
+        // No new dbSNP accessions can be generated, so the variants can only be stored directly using a repository
+        // TODO When the support for new EVA accessions is implemented, this could be changed
+        // In order to do so, replicate the structure of {@link ClusteredVariantRestController}
+        generatedAccessions = repository.save(Arrays.asList(entity1, entity2));
+
+        System.out.println("Generated accessions = " + generatedAccessions);
+    }
+
+    @After
+    public void tearDown() {
+        repository.deleteAll();
+    }
+
+    @Test
+    public void testGetVariantsRestApi() {
+        String identifiers = StreamUtils.createStreamFromIterator(generatedAccessions.iterator())
+                .map(acc -> acc.getAccession().toString()).collect(Collectors.joining(","));
+        String getVariantsUrl = URL + identifiers;
+
+        ResponseEntity<List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>>>
+                getVariantsResponse =
+                testRestTemplate.exchange(getVariantsUrl, HttpMethod.GET, null,
+                                          new ParameterizedTypeReference<
+                                                  List<
+                                                          AccessionResponseDTO<
+                                                                  ClusteredVariant,
+                                                                  IClusteredVariant,
+                                                                  String,
+                                                                  Long>>>() {
+                                          });
+        assertEquals(HttpStatus.OK, getVariantsResponse.getStatusCode());
+        assertEquals(2, getVariantsResponse.getBody().size());
+        assertDefaultFlags(getVariantsResponse.getBody());
```

Comment:
Can we add an assertion to check that the date is properly handled? I deployed this code and the date shows incorrectly as null